massa-models/src/denunciation.rs

```diff
@@ -236,10 +236,10 @@ impl Denunciation {
             }
         };
 
-        Ok(hash_1 != hash_2
+        hash_1 != hash_2
             && signature_1 != signature_2
```

Comment:
`signature_1 != signature_2` is not necessary

Reply:
Removed